client/src/partials/patients/documents/modals/documents.modal.html

```diff
@@ -0,0 +1,70 @@
+<div class="modal-header">
+  <span class="glyphicon glyphicon-paperclip"></span>
+  {{ 'PATIENT_DOCUMENT.ADD' | translate }} {{ 'PATIENT_DOCUMENT.DOCUMENTS' | translate }}
+  <span class="label label-primary"> {{ $ctrl.patient.abbr + $ctrl.patient.reference }}</span>
+</div>
+
+<div class="modal-body">
+  <form class="form" name="documentForm" novalidate>
+
+    <div class="form-group">
+      <label class="require" for="title">{{ 'FORM.LABELS.TITLE' | translate }}</label>
+      <input
+        class="form-control"
+        type="text"
+        name="title"
+        ng-model="$ctrl.title"
+        required>
+    </div>
+
+    <!-- thumbnail  -->
+    <img class="document-upload-thumbnail" ngf-src="$ctrl.file || 'assets/placeholder.gif'">
```

Comment:
Is it possible to set this assignment in the controller?  It would be cleaner there.